docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,234 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases`
```

Comment:
Same as here: https://github.com/elastic/kibana/pull/127686#discussion_r829863031

Reply:
Done in 0cfb1fab28ae261de027ab7416387891fbfe1e26